corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -130,8 +130,8 @@ function parse_meta(type, style) {
  */
 function Container(json) {
     var self = this;
+    self.pubsub = new ko.subscribable();
```

Comment:
This is a weird event aggregator, but I couldn't find an existing one in our dependencies.

Reply:
Interesting. Looks reasonable, I'm definitely happier to use this than to add a new dependency.